### DISCUSSION TITLE
Pythonコードブロックでmatch,case文が使えない

### DISCUSSION THREAD

**Discussion by Cheesecake2960** [increments/qiita-discussions#948] [要望・アイデア・不具合]
Markdownのコードブロック内でPythonのmatch,case文を使用すると
文字色が変わりません。

**Comment by tomoasleep** [comment]
@Cheesecake2960 
ありがとうございます！

Qiita のシンタックスハイライトには [Rouge](https://github.com/rouge-ruby/rouge) というオープンソースのライブラリを使用しています。
調査したところ Rouge 側が Python の match や case 文に未対応のようです。

https://rouge.jneen.net/v4.5.1/python/bWF0Y2ggKDEsIDIpOgogICAgY2FzZSAoMSwgeSk6CiAgICAgICAgcHJpbnQoJ0Nhc2UgMScpCiAgICBjYXNlIF86CiAgICAgICAgcHJpbnQoJ0FueXRoaW5nJyk

なので、この問題の対応には Rouge 側にパッチを送って修正していただく必要があります。今後 Rouge 側にパッチを送ることも検討いたしますが、対応に時間がかかってしまうことに関しましてはご理解いただけますと幸いです :bow:
